paper-checkbox.html

```diff
@@ -164,12 +170,17 @@
         position: relative;
         display: inline-block;
         vertical-align: middle;
-        padding-left: 8px;
+        padding-left: var(--paper-checkbox-label-spacing, 8px);
         white-space: normal;
         pointer-events: none;
         color: var(--paper-checkbox-label-color, --primary-text-color);
       }
 
+      :host-context([dir="rtl"]) #checkboxLabel {
+        padding-right: var(--paper-checkbox-label-spacing, 8px);
+        padding-left: 0;
```

Comment:
I'm not sure -- do you think RTL should be a different style? It should be just "whatever you're normally doing, but flipped around", and we already don't have a mixin for the label, so adding one just for RTL is maybe a bit unnecessary? WDYT?